rmf_task/src/rmf_task/Request.cpp

```diff
@@ -44,6 +44,16 @@ Request::Request(
   // Do nothing
 }
 
+//==============================================================================
+Request::Request(
```

Comment:
Wow nice catch. We should probably update our unit tests to use this constructor.

Reply:
https://github.com/open-rmf/rmf_task/pull/81/commits/27fd7891b49de9ea1e9953e6761caeb14d5c806e